src/frontend/components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetGeneralTitle/index.tsx

```diff
@@ -0,0 +1,157 @@
+  },
+  title: {
+    defaultMessage: 'General',
+    description:
+      'Title of the widget used for setting live title and activate recording.',
+    id: 'components.DashboardVideoLiveWidgetGeneralTitle.title',
+  },
+  placeholderTitleInput: {
+    defaultMessage: 'Enter title of your live here',
+    description:
+      'A placeholder text indicating the purpose of the input and what it is supposed to received.',
+    id: 'components.DashboardVideoLiveWidgetGeneralTitle.placeholderTitleInput',
+  },
+  updateTitleSuccess: {
+    defaultMessage: 'Title updated.',
+    description: 'Message displayed when title update is successful.',
+    id: 'component.DashboardVideoLiveWidgetGeneralTitle.updateTitleSuccess',
+  },
+  updateTitleBlank: {
+    defaultMessage: "Title can't be blank !",
+    description:
+      'Message displayed when the user tried to enter a blank title.',
+    id: 'component.DashboardVideoLiveWidgetGeneralTitle.updateTitleBlank',
+  },
+  updateTitleFail: {
+    defaultMessage: 'Title update has failed !',
+    description: 'Message displayed when title update failed.',
+    id: 'component.DashboardVideoLiveWidgetGeneralTitle.updateTitleFail',
+  },
+  recordingToggleLabel: {
+    defaultMessage: 'Activate live recording',
+    description:
+      'The label associated to the toggle button reponsible of live recording activation / deactivation.',
+    id: 'components.DashboardVideoLiveWidgetGeneralTitle.liveRecordingToggleLabel',
+  },
+  updateAllowRecordingSuccess: {
+    defaultMessage: 'Recording activation status updated.',
+    description:
+      'Message when recording activation status update is successful.',
+    id: 'component.DashboardVideoLiveWidgetGeneralTitle.updateAllowRecordingSuccess',
+  },
+  updateAllowRecordingFail: {
+    defaultMessage: 'Recording activation status update has failed !',
+    description: 'Message when recording activation status update failed.',
+    id: 'component.DashboardVideoLiveWidgetGeneralTitle.updateAllowRecordingFail',
+  },
+});
+
+interface DashboardVideoLiveWidgetGeneralTitleProps {
+  video: Video;
+}
+
+export const DashboardVideoLiveWidgetGeneralTitle = ({
+  video,
+}: DashboardVideoLiveWidgetGeneralTitleProps) => {
+  const intl = useIntl();
+  const [titleState, setTitleState] = useState(video.title);
+  const [checked, setChecked] = useState(video.allow_recording);
+
+  const titleMutation = useUpdateVideo(video.id, {
+    onSuccess: () => {
+      toast.success(intl.formatMessage(messages.updateTitleSuccess));
+    },
+    onError: (err) => {
+      report(err);
+      toast.error(intl.formatMessage(messages.updateTitleFail));
+      setTitleState(video.title);
+    },
+  });
+
+  const allowRecordingMutation = useUpdateVideo(video.id, {
+    onSuccess: (data: Video) => {
+      setChecked(data.allow_recording);
+    },
+    onError: (err) => {
+      report(err);
+      toast.error(intl.formatMessage(messages.updateAllowRecordingFail));
+    },
+  });
```

Comment:
I think you should use `useUpdateVideo` only once and then in the `onSuccess` callback use the `data` or `variables` arguments to determine what have been updated and do theright action accordingly with what you found. You should the same thing in the `onError` callback

Reply:
I gathered the logic in videoMutation function, as you suggested, but and differentiated processing in `onError`.